01-js-intro-and-jquery/other/1_12/resources/script/script.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+var onClickButton =  function () {
+
+    $('#btnSearch').on('click', onPageReady);
+
+};
+
+var onPageReady = function () {
+    var container = $('#albumList');
+    var onRequestSuccess;
+    var onRequestError;
+    var listLength;
+    var albums;
+    var appendAlbum;
+    var $input = $('#txtArtist').val();
```

Comment:
This is not right, `$('#txtArtist').val();` it's not a jQuery object this is a value. The jQuery object is the `$('#txtArtists')`. Take care of this mistake. Also try to use for the input this kind of names: `txt-artist` instead of `txtArtist`.